### ISSUE TITLE
Contours

### ISSUE THREAD

**Issue by Ashwita2619** [AntSimi/py-eddy-tracker#86]
Hi Antoine,

I am confused with the part how the coutours are selected in the algortithm and stored later as detected anticyclones and cyclones…
So far I have understood how the collection of contours is selected for the observation.
It follows:
              collections --> (save) contour_paths --> (test) --> (1) shape_error --> (2) pixel --> (3) amplitude --> (store) obs

I have another question:
I am using OW for the detection instead of SSH
and it confuses me to see the different number of detections with different ranges of OW
for e.g.: the detection of anticyclones and cyclones is different with range:
(a) -4e-09 : -4e-12, (b) -4e-10 : -4e-12

Do you have any clue for this different number of detections with different range of OW?

**Comment by AntSimi** [comment]
Hi,
In amplitude step you also check if there only one extrema or more if you allow more.

For Okubo Weiss, i never used for eddy detection, only play a little with it:
- Did you use same step in you two range (a) and (b) ?
- How detection are different, which version are better?
- When you compare detection with OW field did you see something incoherent?
- How did you implement eddy detection with OW?

Antoine

**Comment by Ashwita2619** [comment]
1. yes, I used the same step for both (a) and (b)
  I used the step value of 4e-12
where, (a) -4e-09 : -4e-12 has 1002 levels, and (b)  -4e-10 : -4e-12 has 102 levels

2. It is hard to say which version is better.
    the detection is different in terms of count in various regions (almost everywhere in the ocean)
I performed numerous tests in different regions and with different time steps to know at which instance what is detected and what is missing with the other. But this doesn't answer me 'why' there are differences as what actual criterion is helping for number of detections.

3. no, I think I don't see anything incoherent
    the structures are fine. 
but why the detections are different from one range to the another I am unable to understand that.

4.  In order to implement the eddy detection with OW
     we just asked the algorithm to use the contours of OW instead of SSH
